Dockerfile

```diff
@@ -4,11 +4,16 @@ MAINTAINER "The Mineiros.io Team <hello@mineiros.io>"
 ENV TFLINT_VERSION="v0.13.4"
 ENV TFLINT_SHA256SUM="f89113271e50259aac318c05f4e9a9a1b4ec4a59afaa9bb4f36438cbb346757f"
 
-RUN apk add --update bash curl git python3 terraform
+RUN apk add --update bash curl git openssl python3 terraform
 
 RUN pip3 install pre-commit
 
-RUN curl -L -o /tmp/tflint.zip https://github.com/terraform-linters/tflint/releases/download/${TFLINT_VERSION}/tflint_linux_amd64.zip && unzip /tmp/tflint.zip -d /usr/local/bin
+ADD https://github.com/terraform-linters/tflint/releases/download/${TFLINT_VERSION}/tflint_linux_amd64.zip .
+ADD https://github.com/terraform-linters/tflint/releases/download/${TFLINT_VERSION}/checksums.txt .
+RUN sed -i '/.*tflint_linux_amd64.zip/!d' checksums.txt
+RUN sha256sum -cs checksums.txt
```

Comment:
that's a weird check as it's the same source ;) ... if i can manipulate the zip i can also update the checksum.. ;)

Reply:
Hmm, how would you do that normally? In our case, both, the released zip file and the txt containing the checksum are hosted on Github. I haven't seen cases, where people would upload one or another to a different source ?